zerver/tests/test_event_system.py

```diff
@@ -1194,7 +1194,7 @@ def test_queries(self) -> None:
             realm_filters=0,
             realm_linkifiers=0,
             realm_playgrounds=1,
-            realm_user=4,
+            realm_user=5,
```

Comment:
It's a bit annoying that these query counts are growing, but I guess we do have a plan to reduce the queries with a client capability in the future (basically to stop sending the legacy permissions `can_` field)